docker-compose.yml

```diff
@@ -17,8 +18,6 @@ services:
       - REFORM_SERVICE_NAME
       - REFORM_TEAM
       - REFORM_ENVIRONMENT
-    volumes:
-      - ./build/install/spring-boot-template:/opt/app/
```

Comment:
?

Reply:
breaks the docker-compose to properly boot the service - always says that the script under /opt/app does not exist. we're copying over everything anyway - no need to volume it up tbh